roles/ucp/templates/ucp.j2

```diff
@@ -40,12 +40,20 @@ start)
     {% else -%}
     /usr/bin/docker run --rm -t --name ucp \
         -v /var/run/docker.sock:/var/run/docker.sock \
-        -e UCP_ADMIN_USER="admin" -e UCP_ADMIN_PASSWORD="orca" \
+        -e UCP_ADMIN_USER={{ ucp_admin_user }} -e UCP_ADMIN_PASSWORD={{ ucp_admin_password }} \
         docker/ucp join --host-address={{ node_addr }} \
           --image-version={{ ucp_version }} \
           --url="https://{{ service_vip }}:443" \
+    {% if ucp_replicator1_node is defined and node_addr == ucp_replicator1_node_addr -%}
```

Comment:
yes, this is true elsewhere as well..

In current implementation of ansible, only master nodes are meant to be part of the cluster (viz. ucarp, etcd, netmaster, volmaster, ceph-mon, swarm and ucp)  and worker nodes are either proxies (like ectd, swarm, ucp) or not part of active cluster membership at all (netplugin, volplugin, ucarp, ceph)...

In almost all cases, for worker nodes to move to active cluster membership they need to be reconfigured. In cluster-mgr this can be taken care of by the cluster-mgr decommission followed by commission workflow, when we have a more dynamic role assignment there.